### ISSUE TITLE
Move MER related acceptance tests from EE core to MER

### ISSUE THREAD

**Issue by joshfeck** [eventespresso/event-espresso-core#857]
Originally reported by @nerrad in [codebase](https://events.codebasehq.com/projects/event-espresso/tickets/11359):

> I originally added MER related acceptance tests to EE core because at the time there was no public repo for MER so we couldn't run nightlies.  However, that obstacle is now removed and I really should get the MER specific tests into the MER add-on.  Right now, if a test runs that has MER active and that test fails, that means any subsequent tests that run that require MER to be inactive will fail as well because MER doesn't get deactivated at the end of the MER specific test.  It also will make core tests go a bit faster if they don't need to install MER or do the MER specific tests.
> 
> @sethshoultes I estimate about a day to do this switchover.  It's not high priority but it will improve the integrity of our acceptance test automation.

**Comment by nerrad** [comment]
@joshfeck I'm thinking we should just close this ticket.  Our acceptance tests framework isn't really getting used so I don't think it's worth the investment of time to write anymore tests for it (or improve the framework).  I was the only one writing tests and its effectiveness is pretty much lost when it doesn't get used. I suspect its not being used because:

- as much as I tried to simplify things, writing e2e tests is still complicated.
- as much as I tried to make it easier, setting up docker to run the tests is likely a roadblock for some.
- some of the tests are a bit unreliable because of the underlying engines used (the tests get paused or stop working all of a sudden).
- it takes _significant_ time to write acceptance tests. Making the time to write e2e tests frankly hasn't been a priority.

Nobody really seems to even be paying attention to whether existing acceptance tests fail (I gave up monitoring things a while ago).

It's been on my mind to switch to using the same e2e testing engine WP is using Gutenberg which would allow us to inherit all the prep work/utils they've done.  Plus getting started with e2e tests is a bit easier. 

So we can leave our existing e2e tests in codeception as they are for the time being.
